dask/values.yaml

```
@@ -131,6 +131,76 @@ worker:
       podTargetLabels: [] # PodTargetLabels transfers labels on the Kubernetes Pod onto the target.
       metricRelabelings: [] # MetricRelabelConfigs to apply to samples before ingestion.
 
+additional_worker_groups:
+  - worker:
+      name: worker  # Dask worker name.
```

In the helm chart the `values.yaml` file contains the default configuration values. Therefore this change means there would be an additional worker group by default. We probably don't want that.

If you take a look at some of the options in here like `extraArgs` you can see that the default is just set to an empty list `[]` but there is an actual example below that is commented out. This is generally good practice for helm charts.

```suggestion
additional_worker_groups: []  # Additional groups of workers to create
  # - worker:
  #     name: worker  # Dask worker name.
```